client/analytics/report/categories/table.js

```diff
@@ -95,7 +99,7 @@ class CategoriesReportTable extends Component {
 					value: category && category.name,
 				},
 				{
-					display: formatValue( 'number', itemsSold ),
+					display: formatValue( currency, 'number', itemsSold ),
```

Comment:
Why do you have to pass `currency` into `formatValue` if the value isn't a currency value?

Reply:
Good question. I went through much of this refactor and then realized that `formatValue` takes the currency config as its first argument. Turns out what we've been calling currency also has numeric format data like `decimalSeparator`. I decided to leave the naming as currency, but maybe in a follow up I can change it to `numericLocale` or `locale` and include other information as well.

<img width="329" alt="Screen Shot 2020-04-02 at 7 51 54 AM" src="https://user-images.githubusercontent.com/1922453/78175184-f2869e80-74b6-11ea-8e09-4ec7a5fd43b6.png">

Reply:
Ok no worries. Everything else looks great :-)